security/Security.php

```diff
@@ -95,7 +95,7 @@ class Security extends Controller {
 	 *
 	 * @var array|string
```

Comment:
The thing is it _can_ be set to a string, it’s [treated the same as setting `default: 'string'`](https://github.com/kinglozzer/sapphire/blob/patch-1/security/Security.php#L237-L239), so I figured I’d leave it. Though I guess it’s not really a message _set_ if you just set it to a string!